maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/controller/IojAppealControllerTest.java

```diff
@@ -74,10 +84,26 @@ void givenWebClientInternalServerErrorStatus_whenFindIsInvoked_thenInternalServe
         WebClientResponseException webClientResponseException =
                 WebClientTestUtils.getWebClientResponseException(HttpStatus.INTERNAL_SERVER_ERROR);
 
-        when(iojAppealService.find(anyInt())).thenThrow(webClientResponseException);
+        when(orchestrationService.find(anyInt())).thenThrow(webClientResponseException);
 
         String endpoint = ENDPOINT_URL + "/" + LEGACY_APPEAL_ID;
         mvc.perform(buildRequestWithTransactionId(HttpMethod.GET, endpoint, true))
                 .andExpect(status().isInternalServerError());
     }
+
+    @Test
+    void givenValidRequest_whenCreateIsInvoked_thenOkResponseIsReturned() throws Exception {
+        when(orchestrationService.create(any(WorkflowRequest.class))).thenReturn(new ApplicationDTO());
+
+        String requestBody = objectMapper.writeValueAsString(TestModelDataBuilder.buildWorkFlowRequest());
+        mvc.perform(buildRequestWithTransactionIdGivenContent(HttpMethod.POST, requestBody, ENDPOINT_URL, true))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON));
+    }
+
+    @Test
+    void givenInvalidRequest_whenCreateIsInvoked_thenBadRequestResponseIsReturned() throws Exception {
```

Comment:
There is a test for find called `givenWebClientInternalServerErrorStatus_whenFindIsInvoked_thenInternalServerErrorResponseIsReturned`, it would be good if we could add a similar one for create.